src/extension.ts

```diff
@@ -22,7 +22,8 @@ export function activate(context: vscode.ExtensionContext) {
         const templateFolderRelativePath = vscode.workspace
             .getConfiguration("blueprint")
             .get("templatesPath") as string;
-        const templateFolderPath = `${vscode.workspace.rootPath}/${templateFolderRelativePath}`;
+        const normalizedTemplateRelativePath = path.normalize(templateFolderRelativePath);
+        const templateFolderPath = path.join(vscode.workspace.rootPath, normalizedTemplateRelativePath);
```

Comment:
Just trying to understand why we use `vscode.workspace.rootPath` here rather than `directoryPath`?

Reply:
Directory path here means the place that we want to put our result (where we are creating the files).

The templateFolderPath is where our templates folder is located. So we start at the workspace root and append the relative path to the template folder.

So workspace might be `/users/reesemclean/projects/my-project`
templateFolderRelativePath might be `/blueprint-templates`
Combining them would be `/users/reesemclean/projects/my-project/blueprint-templates`

directoryPath might be something like `/users/reesemclean/projects/my-project/src/features/home` just wherever you did `New Template from File` from.